### ISSUE TITLE
Koseven Brand

### ISSUE THREAD

**Issue by neo22s** [koseven/koseven#141]
Edit:

Thanks to @evgenykoptelov for the great work, we will be using

![29709312-878c941e-8994-11e7-8f2e-43ea4add52e8](https://user-images.githubusercontent.com/241725/29997163-aaa03acc-900c-11e7-8aec-d542133fd251.png)

----------------
Hello!

It's about time to have some official colors and a decent logo I think xD

Any designer in the room?

I think KO7 simply is nice...

![ko-logo](https://user-images.githubusercontent.com/241725/29476999-b801d650-8466-11e7-8c58-da224cd16b4e.png)

**Comment by jstrobel** [comment]
I think it would be good if the logo or a variation of it is working square for avatar and thinks like that

**Comment by errotan** [comment]
![koseven_logo_draft](https://user-images.githubusercontent.com/292507/29484430-b1ea2c24-84be-11e7-8df4-09fb6ed3f4c4.png)
Besides the logo, an unique slogan or posy would make the project more memorable.

**Comment by evgenykoptelov** [comment]
Hello! I think the bright colors and modern fonts will be more suitable:<br><br><br>
![logos](https://user-images.githubusercontent.com/18583376/29535429-789912c2-86c2-11e7-8e1c-c9f97e5dd53e.png)

**Comment by neo22s** [comment]
I like the most the redesign of the classic logo! KO7

But we really need voting on this... hahaha

thanks a lot guys!!

**Comment by Sebjugate** [comment]
For what my vote is worth, I like the classic-style KO7 as well.

**Comment by satanch** [comment]
"Redisgn of the classic logo" or "Laconic thin inscriptions" for my opinion.

**Comment by errotan** [comment]
The classic logo looks great!

**Comment by jstrobel** [comment]
For me the first looks great: colors and font! And the combination of two colors, just i am not sure about a gradient. If most of you tendent to the classic one, what about a combination?
Two Colors in the classic logo? The Seven and 7 in the blue from the first? So it gets a little "fresher" look ;)
BUT great work and thanks a lot to you mate.

**Comment by Samnan** [comment]
Classic logo redesign looks good, though the one with alphabets looks a big too long for logo, and the one with KO7 too short :) maybe I am comparing it to the old logo, but that is my feedback.

@evgenykoptelov can you try something that is same like classic logo redesign but looks more compact?

**Comment by evgenykoptelov** [comment]
@Samnan, thanks, I can change the spacing between letters, but the logo will be divided. I think I showed it in too large. Perhaps the logo of a different size looks laconic.<br>
![group 12](https://user-images.githubusercontent.com/18583376/29554793-2a46f2ee-8729-11e7-9935-f6691b51f42f.png)

**Comment by evgenykoptelov** [comment]
@Samnan, thanks, I can change the spacing between letters, but the logo will be divided. I think I showed it in too large. Perhaps the logo of a different size looks laconic.
![group 12](https://user-images.githubusercontent.com/18583376/29554815-3a1be10c-8729-11e7-8042-6a1c89701640.png)

**Comment by evgenykoptelov** [comment]
@Samnan, thanks, I can change the spacing between letters, but the logo will be divided. I think I showed it in too large. Perhaps the logo of a different size looks laconic.
![group 12](https://user-images.githubusercontent.com/18583376/29554827-49cd0b94-8729-11e7-8f27-f11d26506457.png)

**Comment by evgenykoptelov** [comment]
@jstrobel, thanks, it’s interesting. I will add two logos with these colors in the comment :-)

**Comment by evgenykoptelov** [comment]
@jstrobel, Added!

**Comment by uBizzy** [comment]
Here are a few more samples and ideas... I am open to make edits to them

![logo-samples-ko7](https://user-images.githubusercontent.com/24408831/29660953-919d8326-88ba-11e7-81dd-d8c7f8c26ea9.jpg)

![screen_shot_2017-08-23_at_5 05 51_pm](https://user-images.githubusercontent.com/24408831/29660969-98c7dc00-88ba-11e7-9ac4-c73cb9db48bd.png)

**Comment by jstrobel** [comment]
@uBizzy Thanks a lot for your ideas.
For me a simple (or how would you call it) font looks best. at the block of 6 the font at the left button looks cleanest for me. I am not sure about the "logo" right now. I like your idea of mixing the seven with a "element". but is the seven visible enought we we use just this element? 
sorry that i am not clear right now, just what crosses my mind ;)
let's see what others think.

**Comment by uBizzy** [comment]
@jstrobel , thank you for your feedback. This was just a first try, let's wait on other comments and if it's necessary, I can work on them a little bit further.

**Comment by evgenykoptelov** [comment]
@jstrobel, What do you think about the classic version with the new colors? So did I show your idea? :-)

**Comment by jstrobel** [comment]
@evgenykoptelov Where is it? I can't see it :(

**Comment by evgenykoptelov** [comment]
@jstrobel, I added a picture in the comment above ;-)
<br>
![29572655-97c061c0-8765-11e7-9676-9b9114e3f666](https://user-images.githubusercontent.com/18583376/29709271-6249eca6-8994-11e7-8815-bad2b0108b6a.png)

**Comment by evgenykoptelov** [comment]
@jstrobel, I added a picture in the comment above ;-)
<br>
![29572655-97c061c0-8765-11e7-9676-9b9114e3f666](https://user-images.githubusercontent.com/18583376/29709312-878c941e-8994-11e7-8f2e-43ea4add52e8.png)

**Comment by jstrobel** [comment]
Ah sorry, yes. For me flat one looks good. Let's see what other think and what the feedback for @uBizzy are :) Thanks a lot so far.

**Comment by Samnan** [comment]
I would say we lock the flat one and use it. This is good enough and better
than the default one. Later it could be revised, better put more time and
effort on some other important things to get the framework to rock ;)


-----------------------------
Samnan ur Rehman
Consultant / Senior Software Developer
about.me/samnan

**Comment by satanch** [comment]
Flat looks really nice, but I still prefer classic redesign.

**Comment by sicay** [comment]
My vote goes to new color solutions (flat) !

**Comment by neo22s** [comment]
Ok!! so we all agree to use New Colors Solution Flat, right?

I will create new issue to do the new site colors and add the logo.

@evgenykoptelov thanks for the great work! can you please attach in that issue the original of the logos?
